test/test_software_modules/test_topic_filer.py

```diff
@@ -0,0 +1,19 @@
+from openag_brain.software_modules.topic_filter import EWMA
+import numpy as np

```

Comment:
Why are we importing numpy? Also, if we need it, we need to specify it as a dependency in `setup.py`.